README.md

```diff
@@ -63,6 +61,7 @@ SpikeInterface is a Python framework designed to unify preexisting spike sorting
 
 All documentation for SpikeInterface can be found [here](https://spikeinterface.readthedocs.io/en/latest/).
 
+Documentation of current release with OLD api is [here](https://spikeinterface.readthedocs.io/en/stable/).
```

Comment:
@samuelgarcia what about CURRENT. I think that OLD gives the idea that people should not use it (but it's not the case yet)